File: ToDoApp_backend/DB/CategoryMaster.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using ToDoApp_backend.DB.Base;

namespace ToDoApp_backend.DB
{
    public class CategoryMaster : DbBase
    {
        public string CategoryName { get; set; }
        public string Summary { get; set; }
        public int SortNo { get; set; }
    }
}

File: frontend_App/todo-list/types/DB/CategoryMaster.ts
import { DbBase } from '../DbBase'

export interface ICategoryMaster {
  UserId: number
  CategoryName: string
  Summary: string
  SortNo: number
}

export class CategoryMaster extends DbBase implements ICategoryMaster {
  Id = 0
  CreateDate = new Date()
  UpdateDate = new Date()
  UserId = 0
  CategoryName = ''
  Summary = ''
  SortNo = 0
}
